src/uwtools/utils/api.py

```diff
@@ -0,0 +1,34 @@
+"""
+Support for API modules.
```

Comment:
A new module providing support routines for API modules. I considered placing these in `uwtools.utils.file` but wasn't sure that was right. If we find these routines are needed by non-API code, we could move them, but this seemed to me good enough for now.

Reply:
Hrm - yeah, thats fair, I agree. They feel like `file` utilities, but they're really just used in `api` for now so should be fine